### ISSUE TITLE
Generate `--repl` and `--chat` tags automatically

### ISSUE THREAD

**Issue by jeanlucthumm** [TheR1D/shell_gpt#562]
One annoying part of `sgpt` workflow right now is having to come up with conversation tags beforehand. Things like ChatGPT web UI will title the conversations automatically so there's no toil on the user side.

We could auto-gen the tag based on the first prompt. Picturing something like

```sh
$ sgpt --auto --chat "Is tomato really a fruit?"
Autotag: fruit_discussion

Yes, a tomato is scientifically classified as a fruit because it develops from the ovary of a flower...
```

**Comment by ldelossa** [comment]
Yes, `chatblade` does this well. Any convo outside of a 'session' is in some pseudo session. You can always refer to the last pseudo-session by just providing "-l" assumingly for 'last-session'.

**Comment by jeanlucthumm** [comment]
Just checked out `chatblade`. I think that's the move -- default to being in the `temp` repl and then give some option to save it under an actual tag. At this point we have the conversation history to generate the tag and if not the user can still set their own tag
